### PR TITLE
Nhsnlink 532-Record and display submitted date when user submits

### PR DESCRIPTION
**Pull request by arianamihailescu**

### CODE REVIEW

File: web/src/main/angular/app/review/review.component.html [hunk 1]
```diff
@@ -69,7 +69,7 @@
             {{getStatusName(report.docStatus)}}
         </td>
         <td style="min-width: 16rem">
-            {{report.submittedDate | date}}
+            <span [ngbTooltip]="getFormattedSubmittedDate(report.submittedDate)">{{report.submittedDate | date}}</span>
```
Comment (seanmcilvenna) [inline web/src/main/angular/app/review/review.component.html:72]: consider changing this to:
`<span ngbTooltip="{{report.submittedDate | date: 'MMM d, y, h:mm:ss a'}}">...`

Comment (seanmcilvenna) [reply web/src/main/angular/app/review/review.component.html:72]: if this works, would allow you to remove the `getFormattedSubmittedDate()` method.

**Review by seanmcilvenna — CHANGES_REQUESTED** [review]

File: web/src/main/angular/app/review/review.component.html (hunk 1 above)
Comment (arianamihailescu) [reply web/src/main/angular/app/review/review.component.html]: I made this change

File: web/src/main/angular/app/report/report.component.ts [hunk 2]
```diff
@@ -7,6 +7,7 @@ import {NgbModal} from '@ng-bootstrap/ng-bootstrap';
 import {ViewLineLevelComponent} from '../view-line-level/view-line-level.component';
 import {ReportModel} from "../model/ReportModel";
 import {ReportSaveModel} from "../model/ReportSaveModel"
+import {DatePipe} from "@angular/common";
```
Comment (seanmcilvenna) [inline web/src/main/angular/app/report/report.component.ts:10]: Remove reference to `DatePipe`

File: web/src/main/angular/app/report/report.component.ts [hunk 3]
```diff
@@ -24,11 +25,17 @@ export class ReportComponent implements OnInit, OnDestroy {
         public reportService: ReportService,
         public toastService: ToastService,
         private modal: NgbModal,
-        private router: Router) {
+        private router: Router,
+        public datepipe: DatePipe) {
```
Comment (seanmcilvenna) [inline web/src/main/angular/app/report/report.component.ts:29]: Remove reference to `DatePipe`

File: web/src/main/angular/app/report/report.component.ts [hunk 4]
```diff
@@ -105,12 +112,6 @@ export class ReportComponent implements OnInit, OnDestroy {
         return false;
     }
 
-    get isDisabled() {
```
Comment (seanmcilvenna) [inline web/src/main/angular/app/report/report.component.ts:108]: Was this intentional?

File: web/src/main/angular/app/review/review.component.ts [hunk 5]
```diff
@@ -6,6 +6,7 @@ import {Router} from '@angular/router';
 import {ReportDefinitionService} from '../services/report-definition.service';
 import {formatDate} from '../helper';
 import {UserModel} from "../model/user-model";
+import {DatePipe} from '@angular/common';
```
Comment (seanmcilvenna) [inline web/src/main/angular/app/review/review.component.ts:9]: Remove reference to `DatePipe`

File: web/src/main/angular/app/review/review.component.ts [hunk 6]
```diff
@@ -35,7 +36,8 @@ export class ReviewComponent implements OnInit {
     constructor(public authService: AuthService,
                 public reportService: ReportService,
                 private reportDefinitionService: ReportDefinitionService,
-                private router: Router) {
+                private router: Router,
+                public datepipe: DatePipe) {
```
Comment (seanmcilvenna) [inline web/src/main/angular/app/review/review.component.ts:40]: Remove reference to `DatePipe`

File: web/src/main/angular/app/app.module.ts [hunk 7]
```diff
@@ -69,6 +70,7 @@ export const initFactory = (configService: ConfigService, authService: AuthServi
         AuthService,
         ReportService,
         CookieService,
+        DatePipe,
```
Comment (seanmcilvenna) [inline web/src/main/angular/app/app.module.ts:73]: Not sure this is needed now that it's not being referenced in a TS constructor...

File: web/src/main/angular/app/app.module.ts [hunk 8]
```diff
@@ -24,6 +24,7 @@ import {ErrorPageComponent} from './error-page/error-page.component';
 import {ViewLineLevelComponent} from './view-line-level/view-line-level.component';
 import {MedAdminComponent} from "./report/med-admin.component";
 import {NotesComponent} from "./components/notes/notes.component";
+import {DatePipe} from "@angular/common";
```
Comment (seanmcilvenna) [inline web/src/main/angular/app/app.module.ts:27]: Not sure this is needed now that it's not being referenced in a TS constructor...

**Review by seanmcilvenna — CHANGES_REQUESTED** [review]

**Review by seanmcilvenna — APPROVED** [review]

**Pull request merged**